src/strings/en-us.json

```diff
@@ -955,14 +955,14 @@
     "LabelTitle": "Title",
     "LabelTonemappingAlgorithm": "Select the Tone mapping algorithm to use",
     "LabelTonemappingDesat": "Tone mapping desat",
-    "LabelTonemappingDesatHelp": "Apply desaturation for highlights that exceed this level of brightness. The higher the parameter, the more color information will be preserved. This setting helps prevent unnaturally blown-out colors for super-highlights, by (smoothly) turning into white instead. This makes images feel more natural, at the cost of reducing information about out-of-range colors. The recommended and default values are 0 and 0.5.",
+    "LabelTonemappingDesatHelp": "Apply desaturation for highlights that exceed this level of brightness. The higher the parameter, the more color information will be preserved. This setting helps prevent unnaturally blown-out colors for super-highlights, by (smoothly) turning into white instead. This makes images feel more natural, at the cost of reducing information about out-of-range colors. The recommended value is 0.",
```

Comment:
```suggestion
    "LabelTonemappingDesatHelp": "Apply desaturation for highlights that exceed this level of brightness to prevent blown-out colors for super-highlights by smoothing to white. Higher values preserve more color information. The recommended value is 0.",
```

Reply:
0 means no additional desaturation. We should document that

Reply:
@gnattu what would you suggest the string be changed to?

Reply:
Just 0(disable) should be enough